src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -38,6 +38,20 @@ public Recipe(int id, IngredientList ingredients, int productId, String productN
         idCounter++;
     }
 
+    /**
+     * Creates a deleted recipe.
+     */
+    public Recipe(int id, IngredientList ingredients, int productId, String productName,
+        ProductQuantity productQuantity, String description) {
+        this.id = id;
+        this.ingredients = ingredients;
+        this.productId = productId;
+        this.productName = productName;
+        this.productQuantity = productQuantity;
+        this.description = description;
+        this.isDeleted = true;
+    }
+
```

Comment:
this constructor might be misleading, would it be better to modify the existing one?
for instance
```
if (!isDeleted) {
    idCounter++;
}
```
in the existing constructor might make it usable as an equivalent to this one